contracts/reputation/ReputationI.sol

```diff
@@ -22,9 +22,9 @@ interface ReputationI {
         address _validator,
         address _withdrawalAddress
     )
-        external;
+        external returns (uint256);
```

Comment:
Please see dev guideline.
returns should be in new line.

Reply:
Updated